pkgs/applications/networking/instant-messengers/telegram/kotatogram-desktop/default.nix

```
@@ -0,0 +1,56 @@
+{ mkDerivation, lib, fetchFromGitHub, pkgconfig, python3, cmake, ninja
+, qtbase, qtimageformats, enchant, xdg_utils, ffmpeg, openalSoft, lzma
+, lz4, xxHash, zlib, minizip, openssl, libtgvoip, range-v3
+}:
+
+with lib;
+
+mkDerivation rec {
+  pname = "kotatogram-desktop";
+  version = "1.1.5";
+
+  src = fetchFromGitHub {
+    owner = "kotatogram";
+    repo = "kotatogram-desktop";
+    rev = "k${version}";
+    sha256 = "1j5wn3k1mr2c39szmyzm0pf720jmbllcaj40p2ydx0p3lir1s760";
+    fetchSubmodules = true;
+  };
+
+  postPatch = ''
+    substituteInPlace Telegram/lib_spellcheck/spellcheck/platform/linux/linux_enchant.cpp \
+      --replace '"libenchant-2.so.2"' '"${enchant}/lib/libenchant-2.so.2"'
+  '';
+
+  nativeBuildInputs = [ pkgconfig python3 cmake ninja ];
```

please use canonical name
```suggestion
  nativeBuildInputs = [ pkg-config python3 cmake ninja ];
```

---

fixed